client.go

```diff
@@ -26,6 +27,8 @@ type Conn struct {
 type Client struct {
 	Net            string            // if "tcp" a TCP query will be initiated, otherwise an UDP one (default is "" for UDP)
 	UDPSize        uint16            // minimum receive buffer for UDP messages
+	TLS            bool              // enables TLS connection (port 853)
```

Comment:
I thought about someone using TLS with all default parameters, in this case TLSConfig would be nil and the TLS flag true. But anyway, if we use only TLSConfig, the user could just do new(TLS.Config) if he wants TLS with all the default values.